subcommands/keys/tuf_updates_sign_prod_targets.go

```diff
@@ -0,0 +1,84 @@
+package keys
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/exp/slices"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	signCmd := &cobra.Command{
+		Use:   "sign-prod-targets --txid=<txid> --keys=<tuf-root-keys.tgz>",
+		Short: "Sign production targets for your Factory with the offline targets key",
+		Long: `Sign production targets for your Factory with the offline targets key.
+New signatures are staged for commit together with TUF root modifications.
+
+There are 3 use cases when this command comes handy:
+- When you added a new offline TUF targets key,
+  and want to sign your Factory production targets with it.
+- When you increase the TUF targets signature threshold,
```

Comment:
```suggestion
- You increase the TUF Targets signature threshold
```

Reply:
hmm... I've never seen us writing `production targets` as `production Targets`.

I'd have to change quite a few existing Fioctl commands to accommodate this suggestion.
So, I prefer to keep it as is for now.

Reply:
That's fine and a good plan! I already have a task to cleanup the commands, so I will just do it then and keep it consistent!

Reply:
So, I left `targets` as `targets` and applied the remainder of your corrections.